.nuspec/Xamarin.Forms.nuspec

```diff
@@ -85,19 +85,6 @@
     <file src="..\Xamarin.Forms.Platform\bin\$Configuration$\netstandard1.0\Xamarin.Forms.Platform.*pdb" target="lib\netstandard1.0" />
     <file src="..\Xamarin.Forms.Platform\bin\$Configuration$\netstandard1.0\Xamarin.Forms.Platform.*mdb" target="lib\netstandard1.0" />
 
-    <file src="Xamarin.Forms.targets" target="build\netstandard1.0\Xamarin.Forms.targets" />
-    <file src="Xamarin.Forms.props" target="build\netstandard1.0\Xamarin.Forms.props" />
-    <file src="Xamarin.Forms.DefaultItems.targets" target="build\netstandard1.0" />
-    <file src="Xamarin.Forms.DefaultItems.props" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Xamarin.Forms.Build.Tasks.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Xamarin.Forms.Core.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Xamarin.Forms.Xaml.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Mono.Cecil.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Mono.Cecil.Mdb.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Mono.Cecil.Pdb.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\Mono.Cecil.Rocks.dll" target="build\netstandard1.0" />
-    <file src="..\Xamarin.Forms.Build.Tasks\bin\$Configuration$\netstandard2.0\System.CodeDom.dll" target="build\netstandard1.0" />
```

Comment:
we think we don't need this anymore

Reply:
@StephaneDelcroix  if it is possible, we'd better remove netstandard1.0 build in XF.Core, XF.Xaml, XF.Maps projects

Reply:
ns1.0 binaries are still required for back-compatibility with PCL projects.